src/pinns_pde_solve.jl

```diff
@@ -1094,7 +1094,7 @@ function SciMLBase.discretize(pde_system::PDESystem, discretization::PhysicsInfo
 
     chain = discretization.chain
     initθ = discretization.init_params
-    flat_initθ = if (typeof(chain) <: AbstractVector) reshape(initθ, :) else  initθ end
+    flat_initθ = if (typeof(chain) <: AbstractVector) vcat(initθ...) else  initθ end
```

Comment:
```suggestion
    flat_initθ = if (typeof(chain) <: AbstractVector) reduce(vcat,initθ) else  initθ end
```

Reply:
Tim said one can't reduce vectorized expressions on the GPU (which is what caused the error). `vcat(A...)`ing is not ideal (and the GPU also warns that this is slow), but I couldn't figure out a more efficient way to do this). `reshape` would be great, but it doesn't work because initθ can be a Vector{Vector{Float64}}